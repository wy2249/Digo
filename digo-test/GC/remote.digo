func digo_main() void {
    a := []string{"1234", "4234fff", "da4dfa", "dfa4fda"}
    b := []string{"hedfa", "fdafedf", "fdafe", "hello"}
    c := []int{1, 2,3,4,5,6,7}

    future1 := f1(a, b, c)

    f, g, h := await future1

    println("%l", f)
    println("%l", g)
    println("%l", h)
}


async remote func f1(a []string, b []string, c []int) ([]string, []string, []int) {
    return b, a, c
}
