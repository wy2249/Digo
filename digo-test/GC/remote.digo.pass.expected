[hedfa, fdafedf, fdafe, hello]
[1234, 4234fff, da4dfa, dfa4fda]
[1, 2, 3, 4, 5, 6, 7]
